pxemgr/config.go

```diff
@@ -71,6 +71,7 @@ type Network struct {
 	SubnetSize   string `yaml:"subnet_size"`
 	VlanId       string `yaml:"vlan_id"`
 	NetworkModel string `yaml:"network_model"`
+	BondMode     string
```

Comment:
so in config.yaml it will look like `bondmode: lacp` ?

Should we use `BondMode     string yaml:"bond_mode" `?